### PR TITLE
Ensure packages are https

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves NA

Ensures all the package-lock.json are using https not htto.

- ~[ ] I have assigned myself to this PR and the corresponding issues~
- ~[ ] Tests added for new features~
- [ ] Test engineer approval

### CODE REVIEW

**Review by dr3 — DISMISSED** [review]

**Comment by bcmn** [comment]
Is it more important to ask why `http` has sneaked back in? Supposedly #195 already forced every package-lock to `https`. This PR may be fine, but we don't want the same PR every two weeks.

File: packages/components/psammead-image-placeholder/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-image-placeholder",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (dr3) [inline packages/components/psammead-image-placeholder/package.json:3]: Needith a changelog

**Review by dr3 — CHANGES_REQUESTED** [review]

**Comment by pjlee11** [comment]
@bcmn the code which automatically updated http to https was how I spotted this. That said, I'd assume this package-lock was built before the https checking code was merged

**Comment by ChrisBAshton** [comment]
@bcmn the Image Placeholder PR was added around the same time as the http->https fix, so did not have the fixes. We shouldn't need to do this again (or if we do, it should only be for maybe one other package).

**Review by ChrisBAshton — DISMISSED** [review]

**Review by ChrisBAshton — DISMISSED** [review]

File: packages/components/psammead-image-placeholder/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # Psammead Image Placeholder Changelog
 
-| Version | Description |
-|---------|-------------|
+| Version | Description                                                                                                                                    |
+| ------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (dr3) [inline packages/components/psammead-image-placeholder/CHANGELOG.md]:
```suggestion
|---------|-------------|
```

**Comment by bcmn** [comment]
> @bcmn the Image Placeholder PR was added around the same time as the http->https fix, so did not have the fixes. We shouldn't need to do this again (or if we do, it should only be for maybe one other package).

Aha, thanks for explaining. The downside of the "Update branch" button!

Old man Ben yells at cloud

File: packages/components/psammead-image-placeholder/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # Psammead Image Placeholder Changelog
 
-| Version | Description |
+| Version | Description                                                                                                                                    |
```
Comment (pjlee11) [inline packages/components/psammead-image-placeholder/CHANGELOG.md]:
```suggestion
| Version | Description |
```

File: packages/components/psammead-image-placeholder/CHANGELOG.md [hunk 4]
```diff
@@ -2,4 +2,5 @@
 
 | Version | Description |
 |---------|-------------|
+| 0.1.1   | [PR#228](https://github.com/BBC-News/psammead/pull/228) Ensure package-lock is https                                                           |
```
Comment (pjlee11) [inline packages/components/psammead-image-placeholder/CHANGELOG.md]:
```suggestion
 | 0.1.1   | [PR#228](https://github.com/BBC-News/psammead/pull/228) Ensure package-lock is https. |
```

File: packages/components/psammead-image-placeholder/CHANGELOG.md [hunk 5]
```diff
@@ -2,4 +2,5 @@
 
 | Version | Description |
 |---------|-------------|
+ | 0.1.1   | [PR#228](https://github.com/BBC-News/psammead/pull/228) Ensure package-lock is https. |
```
Comment (pjlee11) [inline packages/components/psammead-image-placeholder/CHANGELOG.md]:
```suggestion
 | 0.1.1   | [PR#228](https://github.com/BBC-News/psammead/pull/228) Ensure package-lock is https. |
```

**Review by dr3 — APPROVED** [review]

**Review by radiocontrolled — APPROVED** [review]
🔒

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**